test/server/http/admin_test.cc

```diff
@@ -377,12 +377,14 @@ TEST(PrometheusStatsFormatter, MetricName) {
 }
 
 TEST(PrometheusStatsFormatter, FormattedTags) {
+  // if value has - then it should be replaced
```

Comment:
nit: This comment can be on a single line, start with capital ("If") and end with a period.

Reply:
fixed